Testscripts/Linux/validate-da.sh

```diff
@@ -24,11 +24,17 @@ GetDistro
 
 readonly dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency-agent /etc/opt/microsoft/dependency-agent /lib/microsoft-dependency-agent /etc/init.d/microsoft-dependency-agent)
 
-readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+readonly da_installer="InstallDependencyAgent-Linux64.bin"
```

Comment:
If the path, `./`, is needed part of the time, and is being implied part of the time, then include it in the value here.  The goal of putting data such as this into named constants is to reduce/eliminate an opportunity for a defect where two different uses get out of sync with each other.

Reply:
Done